src/stcal/resample/resample.py

```diff
@@ -813,7 +813,9 @@ def add_model(self, model):
             'ymax': ymax,
         }
 
-        self._driz.add_image(data, **add_image_kwargs)
+        f = (iscale / self.pixel_scale_ratio)**2
```

Comment:
Can you rename `f` to something more descriptive?

Reply:
I'm not very familiar with the code and I don't understand this.  
For flux density images `iscale` is defined as `1/sqrt(pixel_scale_ratio)` so in this case 
`f = 1 / pixel_scale_ratio**3`. For flux units, `f = 1 / pixel_scale_ratio**2`.  And the data is scaled by this. In the pipeline we set `pixel_scale_ratio=1` so it doesn't matter in the default case. How about user supplied scales?